codalab/rest/interpret.py

```diff
@@ -300,17 +300,17 @@ def fetch_interpreted_worksheet(uuid):
     # The bundle_infos for bundles that don't need updating are also None.
     if bundle_uuids:
         for i, block in enumerate(interpreted_blocks['blocks']):
-            if 'bundle_info' not in block:
+            if not ('bundles_spec' in block and 'bundle_infos' in block['bundles_spec']):
                 interpreted_blocks['blocks'][i] = None
             else:
-                if isinstance(block['bundle_info'], dict):
-                    block['bundle_info'] = [block['bundle_info']]
+                if isinstance(block['bundles_spec']['bundle_infos'], dict):
+                    block['bundles_spec']['bundle_infos'] = [block['bundles_spec']['bundle_infos']]
```

Comment:
Why did you change `block['bundle_info']` to `block['bundles_spec']['bundle_infos']`? Can each block now contain multiple bundle infos (which wasn't possible before)?

Reply:
Because the object `block` only contains the field of `bundles_spec`, I believe it is because the code has not been updated during development. 
And yes, each block can contain multiple bundle Infos, since a block can be a table, then it will make sense.